frontend/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "switchboard": "switchboard frontend",
```

Comment:
Little typo
```suggestion
  "name": "switchboard frontend",
```

Reply:
There's actually a bit more than that. the name needs to be a single word, no spaces.
rules about it are defined here: https://docs.npmjs.com/creating-a-package-json-file

Reply:
Amended in 76b3bd2